extensions/wikia/PortableInfobox/styles/PortableInfobox.scss

```diff
@@ -98,6 +98,11 @@ $infobox-section-header-background: mix($color-page, $color-links, 75%);
 		}
 	}
 
+	.item-type-group,
+	.item-type-comparison {
```

Comment:
thise selector is excatly the same .item-type-key-val:not(:last-of-type), maybe we should merge them